content/blog/integrating-istio-and-spire.md

```diff
@@ -0,0 +1,463 @@
+
+![](/img/picture4.png)
+
+**1.4**	 After clicking create cluster, give name and description to your cluster and type of cluster. In  our case we have chosen large type.
+
+![](/img/picture5.png)
+
+**1.5**	 Obtain Kubeconfig for your cluster and launch a Web terminal to access your cluster for further steps in the document. 
+
+From the **Containers** main page, click **Launch Service Console** to launch the HPE Ezmeral Runtime Enterprise. Open Kubectl, which allows you to enter commands to communicate with your cluster.
+
+If Kubectl is not installed, download Kubectl and the HPE Kubectl Plugin from the **Dashboard**.
+
+For more information, see** [Dashboard - Kubernetes Tenant/Project Administrator](https://docs.containerplatform.hpe.com/55/reference/kubernetes/tenant-project-administration/Dashboard__Kubernetes_TenantProject_Administrator.html)** in the HPE Ezmeral Runtime Enterprise documentation.
+
+### Step 2: Install SPIRE
+
+Get the quickstart yaml file using **[this link](https://raw.githubusercontent.com/cxteamtrials/caas-trials-content/main/services/spire/spire-quickstart.yaml)** and copy that into your cluster and apply it by using kubectl.
+
+```shellsession
+kubectl apply -f spire-quickstart.yaml
+```
+
+This will install SPIRE into your cluster, along with two additional components: the SPIFFE CSI Driver and the SPIRE Kubernetes **Controller manager** which facilitate the registration of workloads and establishment of federation relationships.
+
+Verify installation of SPIRE by checking if all pods are running and containers within them are up. Specifically, you are looking for agent and SPIRE server.
+
+**Note:** Number of agents depends on number of nodes you are working with. Here we are working with three worker nodes, so three agents are assigned for each node. 
+
+Use the command given below, and you will get the output as shown.
+
+```shellsession
+k8s-spiffe-integ-master-7j7fh-m67q9:~ kubectl get pods -n spire
+NAME                            READY   STATUS    RESTARTS       AGE
+spire-agent-5tlck               3/3     Running   2 (31d ago)    31d
+spire-agent-gnwbj               3/3     Running   1 (31d ago)    31d
+spire-agent-mghnw               3/3     Running   2 (31d ago)    31d
+spire-server-574474c7dc-42kln   2/2     Running   4 (4d1h ago)   31d
+```
+
+### Step 3: Install Istio
+
+#### Download the latest release:
+
+You can download the latest release using the official Istio repository or just copy the following command, which would do the same for you.
+
+```shellsession
+curl -L https://istio.io/downloadIstio | sh -
+```
+
+For details reach out to **[ISTIO download page](https://istio.io/latest/docs/setup/getting-started/#download)**.
+
+cd into the Istio directory and set the path by command:
```

Comment:
I guess "cd" is a command you are expecting them to do? It might be better to spell it out. At the beginning of the sentence, it kind of looks like a typo because you're using the lower case.

Reply:
I updated it to "Get into the Istio directory and set"